File: Ohtap E Commerce Domain Models/Development/PHP/Objects/CurrencyDetails/CurrencyDetail.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\CurrencyDetail;
use   Ohtap\ECommerce\DomainModels\Objects\Languages;
class CurrencyDetail  implements ICurrencyDetail
{

   // public Language $Language;//Language sınıfını yüklemiyor bak
    public string $ID;
    public string $Name;
    public int $State;
    public int $OrderNo;
    public string $Description;
}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/CurrencyDetails/CurrencyDetail.cs
using Ohtap.ECommerce.Domain.Models.Objects.Languages;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ohtap.ECommerce.Domain.Models.Objects.CurrencyDetails
{
    public class CurrencyDetail : CurrencyDetailBasic, ICurrencyDetail
    {
         ILanguage ICurrencyDetail.Language { get; set; }
    }
}
